app/models/concerns/hostext/search.rb

```diff
@@ -235,17 +235,19 @@ def search_by_os_minor(key, operator, value)
         z ||= 0
         operatingsystem_ids = []
         Operatingsystem.all.find_each do |os|
-          os_y, os_z = os.minor.split('.')
-          os_z ||= 0
-          operator_addition1 = (operator.length == 1) ? "=" : ""
-          operator_addition2 = (operator == "=") ? "=" : ""
-          if os_y.to_i.public_send(operator + operator_addition1, y.to_i)
-            if os_y == y
-              if os_z.to_i.public_send(operator + operator_addition2, z.to_i)
+          unless os.minor.empty?
```

Comment:
wouldn't this be better and save us the additional indentation?
```suggestion
          next if os.minor.empty?
```
Just a nitpick, I know I'm bit of a indentation freak :facepalm: xD

Reply:
done :) thansk! @ezr-ondrej